### ISSUE TITLE
4.1-RC03 fails to resolve crash.connectors.ssh during testing

### ISSUE THREAD

**Issue by countfloyd** [corda/corda#5121]
Trying 'Gradle test' on 4.1-RC03 results in unresolved dependency on non-existent crash.shell artifact:

```
> Task :bno-contracts:compileTestKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':bno-contracts:compileTestKotlin'.
> Could not resolve all files for configuration ':bno-contracts:testCompileClasspath'.
   > Could not resolve com.github.corda.crash:crash.shell:810d2b774b85d4938be01b9b65e29e4fddbc450b.
     Required by:
         project :bno-contracts > net.corda:corda-node-driver:4.1-RC03 > net.corda:corda-test-utils:4.1-RC03 > net.corda:corda-node:4.1-RC03 > net.corda:corda-shell:4.1-RC03
      > Could not resolve com.github.corda.crash:crash.shell:810d2b774b85d4938be01b9b65e29e4fddbc450b.
         > Could not get resource 'http://ci-artifactory.corda.r3cev.com/artifactory/corda-solutions-dev/com/github/corda/crash/crash.shell/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.shell-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'.
            > Could not HEAD 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-solutions-dev/com/github/corda/crash/crash.shell/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.shell-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'. Received status code 409 from server: Conflict
      > Could not resolve com.github.corda.crash:crash.shell:810d2b774b85d4938be01b9b65e29e4fddbc450b.
         > Could not get resource 'https://repo.guild1.com/artifactory/libs-snapshot-local/com/github/corda/crash/crash.shell/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.shell-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'.
            > Could not HEAD 'https://repo.guild1.com/artifactory/libs-snapshot-local/com/github/corda/crash/crash.shell/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.shell-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'. Received status code 409 from server: Conflict
   > Could not resolve com.github.corda.crash:crash.connectors.ssh:810d2b774b85d4938be01b9b65e29e4fddbc450b.
     Required by:
         project :bno-contracts > net.corda:corda-node-driver:4.1-RC03 > net.corda:corda-test-utils:4.1-RC03 > net.corda:corda-node:4.1-RC03 > net.corda:corda-shell:4.1-RC03
      > **Could not resolve com.github.corda.crash:crash.connectors.ssh:810d2b774b85d4938be01b9b65e29e4fddbc450b.**
         > Could not get resource 'http://ci-artifactory.corda.r3cev.com/artifactory/corda-solutions-dev/com/github/corda/crash/crash.connectors.ssh/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.connectors.ssh-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'.
            > Could not HEAD 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-solutions-dev/com/github/corda/crash/crash.connectors.ssh/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.connectors.ssh-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'. Received status code 409 from server: Conflict
      > Could not resolve com.github.corda.crash:crash.connectors.ssh:810d2b774b85d4938be01b9b65e29e4fddbc450b.
         > Could not get resource 'https://repo.guild1.com/artifactory/libs-snapshot-local/com/github/corda/crash/crash.connectors.ssh/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.connectors.ssh-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'.
            > Could not HEAD 'https://repo.guild1.com/artifactory/libs-snapshot-local/com/github/corda/crash/crash.connectors.ssh/810d2b774b85d4938be01b9b65e29e4fddbc450b/crash.connectors.ssh-810d2b774b85d4938be01b9b65e29e4fddbc450b.pom'. Received status code 409 from server: Conflict
```

**Comment by countfloyd** [comment]
Same problem with 4.1-RC02

**Comment by mikehearn** [comment]
This may be an artifact of the way release candidates are pushed.

**Comment by nargas-ritu** [comment]
It's fixed 4.9 onwards.
